internal/pkg/flink/internal/utils/input.go

```diff
@@ -20,6 +20,10 @@ func GetStdin() *term.State {
 }
 
 func GetConsoleParser() prompt.ConsoleParser {
+	if fileInfo, err := os.Stat("/dev/tty"); err != nil || fileInfo.Mode().Perm()&0444 == 0 {
```

Comment:
Should I split this up and specify the error for each situation?

Reply:
splitting them up sounds like a better idea